src/components/panes/ImporterColumnPane.jsx

```diff
@@ -13,7 +13,15 @@ import { resolveSettingsComponent } from '../misc/importer/settings';
 @connect(state => ({ importer: state.importer }))
 @injectIntl
 export default class ImporterColumnPane extends PaneBase {
-    getRenderData() {
+    shouldComponentUpdate(nextProps, nextState) {
+        if(nextProps.importer.tableSet == null) {
+            this.closePane();
```

Comment:
This is fine, but in other cases it might be risky closing the pane when the data does not exist, because depending on the circumstances it might be temporary and the data might (re)appear shortly.

Instead, a more robust way would be to handle the relevant action (in this case `RESET_IMPORT`) in the `view` reducer and remove the pane there. No need to fix this now though, since this case should not be able to result in any bugs due to the special nature of the importer interface. Just an FYI for future reference.

Reply:
Yeah, this was the simple solution that I found did not result in any errors due the pane closing and then trying to update stuff. I was looking into using the RESET_IMPORT reducer, however I did not find a simple way of finding the IDs of the open panes. I realized now that each pane has a type which I could filter on, so this has now been resolved and the view reducer is used instead.